server/src/test/java/com/objectcomputing/checkins/services/opportunities/OpportunitiesControllerTest.java

```diff
@@ -82,144 +82,126 @@ void testMemberCreateAnOpportunities(){
     }
 
     @Test
     void testGetFindByDescription() {
-
         MemberProfile memberProfile = createADefaultMemberProfile();
+        Opportunities opportunitiesResponse = createADefaultOpportunities(memberProfile);
 
-
-        Opportunities opportunitiesResponse  = createADefaultOpportunities(memberProfile);
-
-        final HttpRequest<?> request = HttpRequest.GET(String.format("/?description=%s", opportunitiesResponse.getDescription())).basicAuth(ADMIN_ROLE,ADMIN_ROLE);
+        final HttpRequest<?> request = HttpRequest.GET(String.format("/?description=%s", opportunitiesResponse.getDescription())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
         final HttpResponse<Set<Opportunities>> response = client.toBlocking().exchange(request, Argument.setOf(Opportunities.class));
         assertEquals(Set.of(opportunitiesResponse), response.body());
-        assertEquals(HttpStatus.OK,response.getStatus());
-
+        assertEquals(HttpStatus.OK, response.getStatus());
     }
 
-
     @Test
     void testGetFindByPending() {
-
         MemberProfile memberProfile = createADefaultMemberProfile();
+        Opportunities opportunitiesResponse = createADefaultOpportunities(memberProfile);
 
-
-        Opportunities opportunitiesResponse  = createADefaultOpportunities(memberProfile);
-
-        final HttpRequest<?> request = HttpRequest.GET(String.format("/?pending=%s", opportunitiesResponse.getPending())).basicAuth(ADMIN_ROLE,ADMIN_ROLE);
+        final HttpRequest<?> request = HttpRequest.GET(String.format("/?pending=%s", opportunitiesResponse.getPending())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
         final HttpResponse<Set<Opportunities>> response = client.toBlocking().exchange(request, Argument.setOf(Opportunities.class));
-        assertEquals(Set.of(opportunitiesResponse), response.body());
-        assertEquals(HttpStatus.OK,response.getStatus());
 
+        assertEquals(Set.of(opportunitiesResponse), response.body());
+        assertEquals(HttpStatus.OK, response.getStatus());
     }
 
     @Test
     void testGetFindAll() {
-
         MemberProfile memberProfile = createADefaultMemberProfile();
+        Opportunities opportunitiesResponse = createADefaultOpportunities(memberProfile);
 
-        Opportunities opportunitiesResponse  = createADefaultOpportunities(memberProfile);
-
-        final HttpRequest<Object> request = HttpRequest.GET("/").basicAuth(ADMIN_ROLE,ADMIN_ROLE);
-
+        final HttpRequest<Object> request = HttpRequest.GET("/").basicAuth(ADMIN_ROLE, ADMIN_ROLE);
         final HttpResponse<Set<Opportunities>> response = client.toBlocking().exchange(request, Argument.setOf(Opportunities.class));
 
         assertEquals(HttpStatus.OK, response.getStatus());
-        assertNotNull(response.getContentLength());
-        response.equals(opportunitiesResponse);
+        assertTrue(response.getContentLength() > 0, "response.getContentLength() > 0");
+        assertEquals(Set.of(opportunitiesResponse), response.body());
```

Comment:
This is the thing that I changed here (there are 6 instances of the null check)

The rest is formatting... I wish git diff understood formatting a bit better... it never feels big enough to warrant a separate PR, but when put in here it makes it really hard to see what's going on 😿